### PR TITLE
OGD-536 add validation for taxYear range later than current tax year

### PR DESCRIPTION
**Pull request by edgarjimenez**

### CODE REVIEW

File: app/uk/gov/hmrc/individualsincomeapi/util/Dates.scala [hunk 1]
```diff
@@ -42,6 +42,9 @@ object Dates extends ServicesConfig {
     if (fromTaxYear.startYr < TaxYear.current().startYr - selfAssessmentYearHistory)
```
Comment (tom59) [inline app/uk/gov/hmrc/individualsincomeapi/util/Dates.scala]: need to add error scenario in documentation

Comment (edgarjimenez) [reply app/uk/gov/hmrc/individualsincomeapi/util/Dates.scala]: 👍

File: resources/public/api/conf/P1.0/sandbox/income-sa-employments-request.md [hunk 2]
```diff
@@ -37,6 +37,13 @@
          <p>{ &quot;code&quot; : &quot;INVALID_REQUEST&quot;,<br/>&quot;message&quot; : &quot;Invalid time period requested&quot; }</p></td>
     </tr>
     <tr>
+         <td><p>toTaxYear later than the current tax year</p></td>
+         <td><p>Any valid tax year where toTaxYear is later than the current tax year</p>
```
Comment (tom59) [inline resources/public/api/conf/P1.0/sandbox/income-sa-employments-request.md]: That does not sound right "Any valid tax year where toTaxYear is later than the current tax year"
Shouldnt we say instead: "toTaxYear later than the current tax year"
And the one about should probably be "toTaxYear earlier than fromTaxYear"

Comment (edgarjimenez) [reply resources/public/api/conf/P1.0/sandbox/income-sa-employments-request.md]: 👍

**Pull request merged**